File: Modul-4/Soal 4/PRAK404-2410817120019-NadyaRamadhani.py
print("Pilih Program")
print("1. Penjumlahan")
print("2. Pengurangan")
print("3. Perkalian")
print("4. Pembagian")
print("5. Exit")
pilihan = int(input("Masukkan Pilihan: "))

if pilihan==5:
    print("Terimakasih, telah menggunakan kalkulator NADYARAMADHANI")
elif pilihan<1 or pilihan>5:
    print("Input anda salah, silahkan coba lagi")
else:
    nil1 = float(input("Masukkan Nilai Pertama: "))
    nil2 = float(input("Masukkan Nilai Kedua: "))
    
    if pilihan==1:
        print(f"Hasil penjumlahan antara {nil1:.2f} dengan {nil2:.2f} adalah {nil1+nil2:.2f}")
    elif pilihan==2:
        print(f"Hasil pengurangan antara {nil1:.2f} dengan {nil2:.2f} adalah {nil1-nil2:.2f}")
    elif pilihan==3:
        print(f"Hasil perkalian antara {nil1:.2f} dengan {nil2:.2f} adalah {nil1*nil2:.2f}")
    elif pilihan==4:
        if nil2==0:
            print("Tidak dapat dibagi dengan nol")
        else:
            print(f"Hasil pembagian antara {nil1:.2f} dengan {nil2:.2f} adalah {nil1/nil2:.2f}")

File: Modul-4/Soal 4/PRAK404-2410817120019-NadyaRamadhani.c
#include <stdio.h>
int main(){
    int pilihan;
    float nil1, nil2;

    printf("Pilih Program\n");
    printf("1. Penjumlahan\n");
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. Exit\n");
    printf("Masukkan Pilihan: ");
    scanf("%d", &pilihan);
    if(pilihan==5){
        printf("Terimakasih, telah menggunakan kalkulator NADYARAMADHANI\n");
    }else if(pilihan<1 || pilihan>5){
        printf("Input anda salah, silahkan coba lagi\n");
    }else{
        printf("Masukkan Nilai Pertama: ");
        scanf("%f", &nil1);
        printf("Masukkan Nilai Kedua: ");
        scanf("%f", &nil2);
        if(pilihan==1){
            printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f", nil1, nil2, nil1+nil2);
        }else if(pilihan==2){
            printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f", nil1, nil2, nil1-nil2);
        }else if(pilihan==3){
            printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f", nil1, nil2, nil1*nil2);
        }else if(pilihan==4){
            printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f", nil1, nil2, nil1/nil2);
            if(nil2==0){
                printf("Tidak dapat dibagi dengan nol");
            }
        }
    }

    return 0;
}